Blog_NSNotificationCenter.playground/Contents.swift

```diff
@@ -10,55 +10,38 @@ public extension Notifier where Notification.RawValue == String {
     
     // MARK: - Static Computed Variables
     
-    private static func nameFor(notification: Notification) -> String {
-        return "\(self).\(notification.rawValue)"
+    private static func name(forNotification notification: Notification) -> NSNotification.Name {
```

Comment:
This doesnt align with Swift API guidelines, it should be 
`name(for notification: Notification)`

Reply:
Yes, you're right.